src/main/java/net/juniper/netconf/Device.java

```diff
@@ -273,7 +273,7 @@ private void loadPrivateKey() throws NetconfException {
         try {
             sshClient.addIdentity(pemKeyFile);
         } catch (JSchException e) {
-            throw new NetconfException(String.format("Error parsing the pemKeyFile: %s", e.getMessage()));
+            throw new NetconfException(String.format("Error parsing the pemKeyFile (%s): %s", getPemKeyFile(), e.getMessage()));
```

Comment:
the pem file is the private key, isn't it? I am not sure we should include the contents of the key file in the exception.. we don't want it showing up in a stack trace or log.